CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# 0.1.2 - 2012-02-24
+# 0.1.3 - 2021-03-19
+
+* Added support for publishing an x86_64 RPM package; it does not have shell completion support yet.
```

Comment:
Was there a problem adding the shell completion files? They're present in the Debian package. You can use that as template, although the ultimate destination for shared completion files is probably different on RedHat-based systems.